docs/local-setup/running-testnet-windows.md

```diff
@@ -13,67 +13,74 @@ sidebar_label: Running a Node on Windows
     Then restart your computer.
 2. Go to your Microsoft Store and look for Ubuntu; this is the Ubuntu Terminal instance. Install and launch it.
 3. Now you might be asked for username and password, do not use admin as username.
-4. Your Ubuntu Instance does not have OpenSSL, which you will need to run the node. To download OpenSSL, please run the following commands in the Ubuntu Terminal:
+4. Your Ubuntu Instance need initial before next steps
```

Comment:
This sentence sounds incomplete; could you please clarify @boo50

Reply:
OpenSSL will install in next steps :
5. You need to install OpenSSL, which you will need to run the node. To download OpenSSL, please run the following commands in the Ubuntu Terminal:
    ```sh
    cd /tmp
    wget https://www.openssl.org/source/openssl-1.1.1.tar.gz
    tar xvf openssl-1.1.1.tar.gz
    ```
6. After it finished downloading OpenSSL, run the following commands to install:
    ```sh
    cd openssl-1.1.1
    sudo ./config -Wl,--enable-new-dtags,-rpath,'$(LIBRPATH)'
    sudo make
    sudo make install
    ```
 And to install it, win version of Ubuntu need initial setup, which I recommend to do in step4, before instal OpenSSL.
Because otherwise it will fail and ask to install missed packages.